.studiorc

```diff
@@ -65,6 +67,33 @@ function gocode_generation() {
   )
 }
 
+document "static_tests" <<DOC
+  Run static code tests
+DOC
+function static_tests() {
+  install_if_missing core/go gofmt
+
+  ( cd /src || return 1
+    local gofiles
+    gofiles=$(find ./ -maxdepth 3 -name '*.go')
+    [ -z "$gofiles" ] && return 0
+
+    unformatted=$(gofmt -l $gofiles)
```

Comment:
`gofmt` not only verifies format but also when the code doesn't compile. If you have an error then
you will see the following output:
```
[65][default:/src:0]# static_tests
pkg/formatter/csv_test.go:35:2: expected declaration, found '}'
pkg/formatter/csv_test.go:40:2: expected declaration, found actual
pkg/formatter/csv_test.go:47:2: expected declaration, found cbStatus
pkg/formatter/csv_test.go:188:2: expected declaration, found actual
  hab-studio: Passed!!
```

Could we also verify if there are errors from the Go files?